app/services/ImagePickerService.js

```diff
@@ -0,0 +1,23 @@
+import ImagePicker from 'react-native-image-picker';
+
+export const selectPhotoTapped = () => (
```

Comment:
Please make a component out of this code so we can reuse it in other projects/places of Freemarket too.